### ISSUE TITLE
Daytime videos in the morning/day and nighttime videos in the evening/night

### ISSUE THREAD

**Issue by stagelevel** [theothernt/AerialViews#161]
Hello, please would it be possible implement an option where only daytime videos play in the morning and day and only nighttime videos play in the evening and night.

When I get daytime videos appearing at night, my whole living room lights up brightly which can be quite blinding.

Thank you for your hard work!

**Comment by theothernt** [comment]
Yes, this is something I'm looking into but I'm not quite sure on the timeline just yet.

It looks like the Apple and Community videos support 4 groups - sunrise, day, sunset, night

**Comment by stagelevel** [comment]
> Yes, this is something I'm looking into but I'm not quite sure on the timeline just yet.
> 
> It looks like the Apple and Community videos support 4 groups - sunrise, day, sunset, night

Thank you very much, this is my number one feature request.

**Comment by crowley88** [comment]
Is there any update on this feature as it would be a great addition ?

**Comment by theothernt** [comment]
No update yet. I made a few internal changes in the last couple of releases that caused problems, so once that settles, I will look at this feature.

As well at tagging the Apple and Community videos with sunrise, etc - I will also allow folders of those names to be used in a similar way with local, SMB, etc sources.

I just need to figure out the 'time' issue as I don' think Android knows if it's sunrise, etc

**Comment by crowley88** [comment]
Good to hear your still at it. A note on sunrise I don't know how possible it is but if the app could crosscheck with Google search or weather app they usually show sunrise and sunset times in correlation to current location. In Google search you just type in sunrise and it comes up at the top.

The other option could be to either manually or get chat gpt to make a list of sunrise and sunsets for every date and have the app reference that so that it would even work on local offline. Just an idea but no doubt you will figure a way as you always do

**Comment by theothernt** [comment]
It looks like there is code out there to calculate the values locally (ie. no call to Google, etc), so I'll probably use that if it works well.

**Comment by FunFrog-BY** [comment]
...or for starters we can set up a simple schedule (2 time inputs) and then work from there later.

**Comment by theothernt** [comment]
The next public beta, due next week, will have manual selection of day/night videos.

I hope the add automatic night/day video selection in the next couple of betas.

**Comment by theothernt** [comment]
Aerial Views v1.8.0 has been released on GitHub. It should be released on Google Play in a few days.

**Comment by theothernt** [comment]
Just working on the automatic version of the feature at the moment. 

Question for people - do you use the Weather Overlay and already have your location entered?

Also, for "day", should that also include "sunrise" - and "night" should include "sunset" ?

**Comment by stagelevel** [comment]
> Question for people - do you use the Weather Overlay and already have your location entered?

Personally, I don't use this.

> Also, for "day", should that also include "sunrise" - and "night" should include "sunset" ?

Sound good to me, unless others agree on something better.

**Comment by theothernt** [comment]
Ok, I might have a public beta to test in the next 3 or 4 days.

**Comment by FunFrog-BY** [comment]
> Question for people - do you use the Weather Overlay and already have your location entered?

Yes, always.

> Also, for "day", should that also include "sunrise" - and "night" should include "sunset" ?

It could be configurable.

Can you please make an option for night mode to just play next video and instantly pause it on first frame, when it's black? Because currently my wife is doing it manually every night before bed.
After night pasess it could just resume playing.
